tests/conftest.py

```diff
@@ -1277,12 +1286,39 @@ def pause_cluster_load(request):
     """
     Pause the background cluster load
```

Comment:
I would add some more details about that this fixture is pausing the load and do not resuming. For resume, you can use resume_cluster_load fixture.

Reply:
added more info in previous commit